README.md

```diff
@@ -5,3 +5,10 @@ MSMQ transport for NServiceBus
 NOTE: As Microsoft is not making MSMQ available for .NET Core, building new systems using MSMQ is not recommended. See https://particular.net/blog/msmq-is-dead
 
 Documentation can be found at <https://docs.particular.net/transports/msmq/>.
+
+## How to test locally
+
+* MSMQ needs to be installed and enabled on the current machine.See the [MSMQ configuration documentation](https://docs.particular.net/transports/msmq/?version=msmqtransport_2#msmq-configuration) on how to install MSMQ.
+* A MSSQL Server needs to be accessible either via:
+  * A local SQL Express installation containing a `nservicebus` database.
```

Comment:
do you also need to create the database manually upfront or will SQL Express create it on the fly?

Reply:
This should be created upfront, the [tables will be created](https://github.com/Particular/NServiceBus.Transport.Msmq/blob/328ca2c1863ed20cac20b51e8e8d88644d294cf7/src/NServiceBus.Transport.Msmq/DelayedDelivery/Sql/TimeoutTableCreator.cs), but the database is assumed to be available.